release-doc-in-process/all_vf.TTL

```diff
@@ -128,14 +128,34 @@ vf:action
         rdfs:domain         vf:EventTemplate , vf:Intent , vf:Commitment , vf:Event ;
         rdfs:range          vf:Action .
 
-vf:occursIn
+vf:inputOf
         a                   owl:ObjectProperty ;
         rdfs:domain         vf:EventTemplate , vf:Event ;
         rdfs:range          vf:ProcessTemplate , vf:Process .
 
-vf:willOccurIn
+vf:outputOf
         a                   owl:ObjectProperty ;
-        rdfs:domain         vf:Intent , vf:Commitment ;
+        rdfs:domain         vf:EventTemplate , vf:Event ;
```

Comment:
Please notice that git gives here inaccurate diff, I replaced *vf:occursIn* with two properties *vf:inputOf* and *vf:outputOf*. so they have same domain & range.